rosplane/src/estimator_example.cpp

```diff
@@ -9,6 +9,11 @@ float radians(float degrees)
   return M_PI*degrees/180.0;
 }
 
+double wrap(double fixed_heading, double wrapped_heading) {
```

Comment:
I would replace heading with angle to make the function more general, and maybe make the function name more clear as to how exactly the angles are being wrapped, for example wrap_within_180